### DISCUSSION TITLE
importing problem

### DISCUSSION THREAD

**Discussion by mkogfcgvcdd** [Cyfrin/foundry-full-course-cu#1124] [Q&A]
```solidity
 import "./Simplestorage.sol"
```
Error: not found SimpleStorage.sol

**Comment by 0xusmanf** [comment]
In lesson 6, normally you would do the following to import simple storage in your deploy script and test file. Since in your case location of these files is not mentioned I would suggest follow the paths where Patrick saved these files and use the command below to import.
```solidity
import {SimpleStorage} from "../src/SimpleStorage.sol";
```
Also there is a typo in your import command, it should be `import "./SimpleStorage.sol"`

**Comment by vigneshbalu96** [comment]
Typo issue, it will work as intended if you change it to "SimpleStorage.sol"
